lib/api/v1/health.rb

```diff
@@ -0,0 +1,25 @@
+require "portus/health"
+
+module API
+  module V1
+    class Health < Grape::API
+      version "v1", using: :path
+
+      desc "Ping this API", tags: ["health"], detail: "Returns 200 as a status code"
+      get("_ping") { status 200 }
+
+      resource :health do
```

Comment:
Should this be a resource? This changes the endpoint from `/_health` to `/health`.

Reply:
This was a conscious change: the underscore means "we are not sure about this endpoint or we may change the format even in this same version". I think this shouldn't happen for the health endpoint: if we change the format, it should be in a v2. I believe that the format is nice enough for now.

As for the `/_ping`, I'm still not convinced about it, since I think the health endpoint is enough, but at the same time I think it's nice to have a minimal endpoint confirming the reachability of the server.

Reply:
Fair enough! 👍